contracts/KyberWrapper.sol

```diff
@@ -0,0 +1,161 @@
+pragma solidity 0.4.18;
+
+
+import "./ERC20Interface.sol";
+import "./Withdrawable.sol";
+import "./Utils.sol";
+import "./PermissionGroups.sol";
+import "./KyberReserveInterface.sol";
+import "./KyberNetworkInterface.sol";
+
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////
+/// @title Kyber Network Wrap main contract
+contract KyberWrapper is Withdrawable, Utils {
+
+    KyberNetworkInterface public kyberNetworkContract;
+    uint                  public maxGasPrice = 50 * 1000 * 1000 * 1000; // 50 gwei
```

Comment:
not sure it should be configurable here. just make a function that returns it from real network.